crates/bevy_winit/src/lib.rs

```diff
@@ -63,10 +63,10 @@ use crate::web_resize::{CanvasParentResizeEventChannel, CanvasParentResizePlugin
 #[cfg(target_os = "android")]
 pub static ANDROID_APP: std::sync::OnceLock<AndroidApp> = std::sync::OnceLock::new();
 
-/// A [`Plugin`] that uses [`winit`] to create and manage windows, and receive window and input
```

Comment:
Why remove the crate-level links? I find them useful.

Reply:
They work on locally generated versions of the doc (by running `cargo doc --open`) which is generally more useful, since the search bar can search for docs on more than a single dependency.

But I'm OK with removing them if they don't works on docs.rs